src/main/kotlin/com/sourcegraph/cody/history/HistoryService.kt

```diff
@@ -122,12 +136,16 @@ class HistoryService(private val project: Project) :
   }
 
   private fun getOrCreateChat(internalId: String): ChatState {
-    val found = state.chats.find { it.internalId == internalId }
-    if (found != null) return found
-    val activeAccountId = CodyAuthenticationManager.getInstance(project).getActiveAccount()?.id
-    val newChat = ChatState.create(activeAccountId, internalId)
-    state.chats += newChat
-    return newChat
+    val accountHistory = getOrCreateActiveAccountHistory()
+    val found = accountHistory.chats.find { it.internalId == internalId }
+    return found ?: ChatState.create(internalId).also { accountHistory.chats += it }
+  }
+
+  private fun getOrCreateActiveAccountHistory(): AccountHistoryState {
+    val activeAccountId = CodyAuthenticationManager.getInstance(project).getActiveAccount()!!.id
```

Comment:
Please try to never use `!!`.
Either `getActiveAccount` should not return `null` (and then we should change it) or `null` is a valid state (or marker for error) and then we should handle it properly.

Reply:
Fixed